wp-highlight-user.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * Plugin Name: WP Hightlight User
+ * Plugin URI: http://bonnierpublications.com
+ * Description: adds a checkbox to all users to have the option highlight them
+ * Version: 0.1
+ * Author: Bonnier
+ * Author URI: http://bonnierpublications.com
+ */
+
+namespace WpHighlightUser;
+
+
+if (!defined('ABSPATH')) {
+    exit;
+}
+
+// Handle autoload so we can use namespaces
+spl_autoload_register(function ($className) {
+
+    if (strpos($className, __NAMESPACE__) !== false) {
+        $className = str_replace("\\", DIRECTORY_SEPARATOR, $className);
+        //require_once(__DIR__ . DIRECTORY_SEPARATOR . $className . '.php');
+        require_once(__DIR__ . DIRECTORY_SEPARATOR . Plugin::CLASS_DIR . DIRECTORY_SEPARATOR . $className . '.php');
+    }
+});
+
+class Plugin
+{
+    /**
+     * Text domain for translators
+     */
+    const TEXT_DOMAIN = 'wp-highlight-user';
+    const CLASS_DIR = 'src';
+
+    /**
+     * @var object Instance of this class.
+     */
+    private static $instance;
+
+    private $settings;
+
+    /**
+     * @var string Filename of this class.
+     */
+    public $file;
+
+    /**
+     * @var string Basename of this class.
+     */
+    public $basename;
+
+    /**
+     * @var string Plugins directory for this plugin.
+     */
+    public $pluginDir;
+
+    /**
+     * @var Object
+     */
+    public $scripts;
```

Comment:
I guess this an unused property. Probably a copy/past thing ? just clean up if your not using it :)